flask_security/core.py

```diff
@@ -494,18 +511,9 @@ def init_app(self, app, datastore=None, register_blueprint=True,
 
         identity_loaded.connect_via(app)(_on_identity_loaded)
 
-        state = _get_state(app, self.datastore,
-                           login_form=login_form,
-                           confirm_register_form=confirm_register_form,
-                           register_form=register_form,
-                           forgot_password_form=forgot_password_form,
-                           reset_password_form=reset_password_form,
-                           change_password_form=change_password_form,
-                           send_confirmation_form=send_confirmation_form,
-                           passwordless_login_form=passwordless_login_form,
-                           anonymous_user=anonymous_user)
-
-        if register_blueprint:
+        self._state = state = _get_state(app, datastore, **kwargs)
+
+        if kwargs['register_blueprint']:
```

Comment:
IMHO we should keep `register_blueprint` in arguments since it is not really a state.